packages/common/src/sites/HubSite.ts

```diff
@@ -424,6 +429,9 @@ export class HubSite
       editor
     );
 
+    const followersGroup = await this.getFollowersGroup();
+    setProp("_followers.isDiscussable", isDiscussable(followersGroup), editor);
```

Comment:
Not sure if this is a valid edge case, but I'm curious if this will be problematic if the followers group doesn't exist or is inaccessible?